### PR TITLE
PMM-10286 Misleading message

### PR DESCRIPTION
**Pull request by rnovikovP**
Removed recommendation for IA and  added alternative options

changes based on https://docs.google.com/document/d/1Xy8pf-gtABst5m_YnNK69M7xjQVgLPCOZ4AMZN0x3T8/edit

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -99,18 +99,7 @@ The default values (in seconds):
...
-
-Alerting in Grafana allows attaching rules to your dashboard panels.  Grafana Alerts are already integrated into PMM Server and may be simpler to get set up.
-
-Alertmanager allows the creation of more sophisticated alerting rules and can be easier to manage installations with a large number of hosts. This additional flexibility comes at the expense of simplicity.
-
-We only offer support for creating custom rules to our customers, so you should already have a working Alertmanager instance prior to using this feature.
-
-!!! seealso alert alert-info "See also"
-    [PMM Alerting with Grafana: Working with Templated Dashboards](https://www.percona.com/blog/2017/02/02/pmm-alerting-with-grafana-working-with-templated-dashboards/)
+Please see the availible [Alerting Options](/using/alerting.html#alternative-alerting-options)
```
Comment (catalinaadam) [inline docs/faq.md:102]:
```suggestion
Check out the available [PMM Alerting Options](/using/alerting.html#alternative-alerting-options). 
```

File: docs/using/alerting.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Integrated Alerting
 
 !!! caution alert alert-warning "Disclaimer"
-    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. Therefore, we recommend that early adopters use Integrated Alerting for testing purposes only.
+    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other cases, please see  [Alternative alerting options](#alternative-alerting-options) section
```
Comment (catalinaadam) [inline docs/using/alerting.md:4]:
```suggestion
    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other cases, see the [Alternative alerting options](#alternative-alerting-options) section.
```

File: docs/using/alerting.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Integrated Alerting
 
 !!! caution alert alert-warning "Disclaimer"
-    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. Therefore, we recommend that early adopters use Integrated Alerting for testing purposes only.
+    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other cases, see the [Alternative alerting options](#alternative-alerting-options) section.
```
Comment (rasika-chivate) [inline docs/using/alerting.md:4]:
```suggestion
    Integrated Alerting is currently in [Technical Preview](../details/glossary.md#technical-preview). This means that this feature is still under development and is subject to change. For other alerting options, see [Alternative alerting options](#alternative-alerting-options).
```

File: docs/using/alerting.md [hunk 4]
```diff
@@ -170,8 +170,10 @@ To silence alerts:
 To reactivate silenced alerts, click **Unsilence All** or click on the disabled bell icon <i class="uil uil-bell-slash"></i> corresponding to the alerts you want to reactivate.
 
 ## Alternative alerting options
-PMM Integrated Alerting was introduced as an alternative to Grafana Alerting and Prometheus’ Alertmanager.
+| Use | <i class="uil uil-thumbs-up"></i> **Benefits** | <i class="uil uil-thumbs-down"></i> **Drawbacks**
```
Comment (rasika-chivate) [inline docs/using/alerting.md:173]:
```suggestion
| Use | <i class="uil uil-thumbs-up"></i> **Pros** | <i class="uil uil-thumbs-down"></i> **Cons**
```

File: docs/using/alerting.md [hunk 5]
```diff
@@ -170,8 +170,10 @@ To silence alerts:
...
+|---|---|---
+| [External Alertmanager](/how-to/configure.html#alertmanager-integration) | 1. A way to integrate PMM when a company has an existing [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). | 1.  For experts who use it already
```
Comment (rasika-chivate) [inline docs/using/alerting.md:175]:
```suggestion
| [External Alertmanager](/how-to/configure.html#alertmanager-integration) | A way to integrate PMM when an organization has an existing [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). | For experts who already use it
```

File: docs/using/alerting.md [hunk 6]
```diff
@@ -170,8 +170,10 @@ To silence alerts:
 To reactivate silenced alerts, click **Unsilence All** or click on the disabled bell icon <i class="uil uil-bell-slash"></i> corresponding to the alerts you want to reactivate.
 
 ## Alternative alerting options
-PMM Integrated Alerting was introduced as an alternative to Grafana Alerting and Prometheus’ Alertmanager.
+| Use | <i class="uil uil-thumbs-up"></i> **Benefits** | <i class="uil uil-thumbs-down"></i> **Drawbacks**
+|---|---|---
+| [External Alertmanager](/how-to/configure.html#alertmanager-integration) | 1. A way to integrate PMM when a company has an existing [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/). | 1.  For experts who use it already 
+| **Recommended**<br>[Native Grafana Alerting](https://grafana.com/docs/grafana/v8.5/alerting/) | 1. Build-in<br>2. Feature-rich. | 1. Available in the latest PMM versions only.
```
Comment (rasika-chivate) [inline docs/using/alerting.md:176]:
```suggestion
| [Native Grafana Alerting](https://grafana.com/docs/grafana/v8.5/alerting/) | 1. Built-in<br>2. Feature-rich. | Available in the latest PMM versions only.
```

Comment (rasika-chivate) [reply docs/using/alerting.md:176]: Is it a good idea to recommend Native Grafana Alerting as we want to promote Integrated Alerting...

File: docs/using/alerting.md [hunk 7]
```diff
@@ -170,8 +170,10 @@ To silence alerts:
...
+| [Integrated Alerting](#) | 1. Simple. | 1. Technical Preview.<br>2. Less integrations then in _Native Grafana Alerting_.
```
Comment (rasika-chivate) [inline docs/using/alerting.md:177]:
```suggestion
| [Integrated Alerting](#) | Simple. | 1. Technical Preview.<br>2. Less integrations as compared to _Native Grafana Alerting_.
```

File: docs/using/alerting.md [hunk 8]
```diff
@@ -170,8 +170,10 @@ To silence alerts:
...
+| [Legacy Grafana Alerting](https://grafana.com/docs/grafana-cloud/legacy-alerting/) | N/A | 1. Deprecated.
```
Comment (rasika-chivate) [inline docs/using/alerting.md:178]: If it's deprecated, why are we mentioning it here...

**Review by rasika-chivate — CHANGES_REQUESTED** [review]

File: docs/using/alerting.md (hunk 6 above)
Comment (rnovikovP) [reply docs/using/alerting.md:176]: no,   in 2.31 we;ll merge  Integrated Alerting and Native Grafana alerting and  -  Native Grafana alerting  will be the base for this feature

**Comment by catalinaadam** [comment]
> @rasika-chivate can you please re-review after @rnovikovP 's comments? This has been open for too long now. Not sure if this should be merged to main, though. Probably to your release branch.

@fabio-silva, could we leave it open until PMM 2.31? I need to know the final name for alerting and other info to be able to review this.

**Comment by fabio-silva** [comment]
> > @rasika-chivate can you please re-review after @rnovikovP 's comments? This has been open for too long now. Not sure if this should be merged to main, though. Probably to your release branch.
> 
> @fabio-silva, could we leave it open until PMM 2.31? I need to know the final name for alerting and other info to be able to review this.

Sure, makes sense. Shouldn't the base branch be other, though?

**Comment by rasika-chivate** [comment]
> > > @rasika-chivate can you please re-review after @rnovikovP 's comments? This has been open for too long now. Not sure if this should be merged to main, though. Probably to your release branch.
> > 
> > 
> > @fabio-silva, could we leave it open until PMM 2.31? I need to know the final name for alerting and other info to be able to review this.
> 
> Sure, makes sense. Shouldn't the base branch be other, though?

Yes. The base branch should be different.

**Comment by rnovikovP** [comment]
@rasika-chivate, @catalinaadam, I would rather see this merged, and for  2.31, we'll correct the statement based on the 2.31 release and naming. 
But keeping the old information on our Doc - leads to more confused users.

**Comment by rasika-chivate** [comment]
Integrated alerting doesn't exist anymore. It has been replaced by Percona Alerting so closing the PR...

**Pull request closed without merging**